WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2231,10 +2231,15 @@ static GSourceFuncs _handlerIntervention =
 
             // Allow mixed content.
             bool enableWebSecurity = _config.Secure.Value();
+#if WEBKIT_CHECK_VERSION(2, 38, 0)
```

Comment:
Dont you think it is better to check here as "greater than" 2.38 iso "equal"?

Reply:
Correction: 
I mean >= 2.38

Reply:
It checks for >= 2.38.

https://github.com/WebPlatformForEmbedded/WPEWebKit/blob/362d5b37f9f72286b906c40e2785d351dcce6eac/Source/WebKit/UIProcess/API/wpe/WebKitVersion.h.in#L67